fixtures/has-yarn-lock/yarn.lock

```diff
@@ -2,14 +2,19 @@
 # yarn lockfile v1
 
 
-dep-of-pkg-with-1-dep@^1.0.0, dep-of-pkg-with-1-dep@^101.0.0:
-  version "1.1.0"
-  resolved "https://registry.nlark.com/dep-of-pkg-with-1-dep/download/dep-of-pkg-with-1-dep-1.1.0.tgz#67d3ac196cb08162e543b87b3b1829c76072c927"
-  integrity sha1-Z9OsGWywgWLlQ7h7Oxgpx2ByySc=
+dep-of-pkg-with-1-dep@^100.0.0:
+  version "100.0.0"
+  resolved "https://registry.nlark.com/dep-of-pkg-with-1-dep/-/dep-of-pkg-with-1-dep-100.0.0.tgz#600117cbae0b97475382c8f20be93bba066eee0f"
+  integrity sha512-SHFc2UqTM7Vkk6n1stpLu3xRIBJniuiHhByk2ZXJ9RGepHH6WUbRlxRvhGPpIVYfcvtcag9NGPtoS/zTN2fnjA==
+
+dep-of-pkg-with-1-dep@^101.0.0:
+  version "101.0.0"
+  resolved "https://registry.nlark.com/dep-of-pkg-with-1-dep/-/dep-of-pkg-with-1-dep-101.0.0.tgz#3214a596874c093513f10766033112e05839fe7f"
+  integrity sha512-t0/gFM29VnZhyyDu0ay/u42e8tPi2lkJtdbBXdUujfFFeXxvSJXISpddbVRwjNkKSX6fUROMek29b9b+MqYDxA==
 
 pkg-with-1-dep@*:
-  version "1.0.0"
```

Comment:
I cleaned up this test case. 1.0.0 wasn't a real version in the [registry mock](https://github.com/pnpm/registry-mock/tree/main/packages).